Branch-SDK/src/main/java/io/branch/referral/TrackingController.java

```diff
@@ -11,7 +13,7 @@
  * </p>
  */
 
-class TrackingController {
+public class TrackingController {
```

Comment:
I don't think making this public makes any difference because only `isTrackingDisabled` method is public, which is public in `Branch.java` as well. The method `isTrackingDisabled` was needed to avoid a potential NPE.